debian/copyright

```diff
@@ -3,7 +3,7 @@ Upstream-Name: game.shader.presets
 Source: https://github.com/kodi-game/game.shader.presets
 
 Files: *
-Copyright: 2014-2020 Team Kodi
+Copyright: 2014-2021 Team Kodi
```

Comment:
Why not 2023 here?

Reply:
I copied from game.libretro without much brain juice going into it. Thanks for catching, will update.